amoro-common/src/main/java/org/apache/amoro/table/TableMetaStore.java

```diff
@@ -179,6 +181,7 @@ private TableMetaStore(Configuration configuration) {
     this.krbPrincipal = null;
     this.accessKey = null;
     this.secretKey = null;
+    this.configuration = configuration;
```

Comment:
Why not just create a `RuntimeContext` here and set it. So you don't need to modify `getRuntimeContext()`.
The current code is OK with me, just my humble take.

Reply:
got it, updated, thanks